src/meta/model_v2/src/actor.rs

```diff
@@ -56,6 +56,7 @@ pub struct Model {
     pub parallel_unit_id: i32,
     pub upstream_actor_ids: ActorUpstreamActors,
     pub vnode_bitmap: Option<VnodeBitmap>,
+    pub expr_context: Option<ExprContext>,
```

Comment:
Could you elaborate a little bit on why it is preferable (or necessary) to make this optional? When would it be `None`?

Reply:
Reviewed again and removed the Option.